azure-pipelines.yml

```diff
@@ -11,9 +11,12 @@ jobs:
     vmImage: ubuntu-16.04
   strategy:
     matrix:
+      Ubuntu2004:
```

Comment:
Nice to get this in so quick after release.

You must update Readme to reflect this change.